crates/vm/levm/src/vm.rs

```diff
@@ -617,6 +617,8 @@ impl VM {
                 .ok_or(VMError::Internal(InternalError::UndefinedState(-1)))?,
         );
         // "The max refundable proportion of gas was reduced from one half to one fifth by EIP-3529 by Buterin and Swende [2021] in the London release"
+        // We update the report to the actual gas refunded so that the diff in the test reports is correct
```

Comment:
```suggestion

```
This comment is not necessary, maybe it can be changed or deleted. Remember that the VM is not only used for the EFTests.